support-frontend/assets/helpers/productCatalog.ts

```diff
@@ -103,7 +103,7 @@ const addFreeBenefit = {
 };
 
 const newsletterBenefit = {
```

Comment:
Should this be renamed since it doesn't relate to any newsletter benefit now?

Reply:
Ok, renamed to `supportBenefit` now